### ISSUE TITLE
Find proper event for window drawing.

### ISSUE THREAD

**Issue by harrisi** [harrisi/Console#57]
It is possible that it will only be necessary to draw when information on the screen is updated as there is no physics loop.

**Comment by harrisi** [comment]
I believe reducing the number of drawing occasions would help with #34.

**Comment by ghost** [comment]
That is true. It seems the only reason to update the screen will be due to a change in the displayed glyphs.
